### PR TITLE
Adding inprogress-circle and notstarted-circle icons for manager dash…

### PR DESCRIPTION
**Pull request by jlai-d2l**
…board.

### CODE REVIEW

File: tier1-icons.html [hunk 1]
```diff
@@ -40,8 +40,7 @@
           <path d="M17.707 5.707l-8 8a1 1 0 0 1-1.414 0l-8-8A1 1 0 0 1 1 4h16a1 1 0 0 1 .924.617A.97.97 0 0 1 18 5a1 1 0 0 1-.293.707z"/>
         </g>
         <g id="arrow-expand-small" mirror-in-rtl="true">
-          <path d="M6,14V6c0-0.4,0.2-0.8,0.6-0.9S7.4,5,7.7,5.3l4,4c0.4,0.4,0.4,1,0,1.4l-4,4C7.4,15,7,15.1,6.6,14.9
-	C6.2,14.8,6,14.4,6,14z M8,8.4v3.2L9.6,10L8,8.4z"/>
+          <path d="M6,14V6c0-0.4,0.2-0.8,0.6-0.9S7.4,5,7.7,5.3l4,4c0.4,0.4,0.4,1,0,1.4l-4,4C7.4,15,7,15.1,6.6,14.9&#xA;&#x9;C6.2,14.8,6,14.4,6,14z M8,8.4v3.2L9.6,10L8,8.4z"/>
```
Comment (dbatiste) [inline tier1-icons.html]: Was this meant to change?

File: tier3-icons.html [hunk 2]
```diff
@@ -136,11 +142,8 @@
           <path d="M7,2,25,15,7,28V2M7,0A2,2,0,0,0,5,2V28a2,2,0,0,0,3.17,1.62l18-13a2,2,0,0,0,0-3.24l-18-13A2,2,0,0,0,7,0Z"/>
         </g>
         <g id="play">
-          <path d="M26,2c1.103,0,2,0.897,2,2v22c0,1.103-0.897,2-2,2H4c-1.103,0-2-0.897-2-2V4c0-1.103,0.897-2,2-2H26 M26,0H4
-	C1.8,0,0,1.8,0,4v22c0,2.2,1.8,4,4,4h22c2.2,0,4-1.8,4-4V4C30,1.8,28.2,0,26,0L26,0z"/>
-          <path d="M12,9l9,6l-9,6V9 M12,7c-0.324,0-0.648,0.078-0.944,0.237C10.406,7.585,10,8.262,10,9v12
-	c0,0.738,0.406,1.415,1.056,1.763C11.352,22.922,11.676,23,12,23c0.388,0,0.775-0.113,1.11-0.336l9-6C22.666,16.293,23,15.669,23,15
-	s-0.334-1.293-0.891-1.664l-9-6C12.775,7.113,12.388,7,12,7L12,7z"/>
+          <path d="M26,2c1.103,0,2,0.897,2,2v22c0,1.103-0.897,2-2,2H4c-1.103,0-2-0.897-2-2V4c0-1.103,0.897-2,2-2H26 M26,0H4&#xA;&#x9;C1.8,0,0,1.8,0,4v22c0,2.2,1.8,4,4,4h22c2.2,0,4-1.8,4-4V4C30,1.8,28.2,0,26,0L26,0z"/>
```
Comment (dbatiste) [inline tier3-icons.html]: Same with these.  These are line feed characters.  Do we care if these are here?

File: tier1-icons.html (hunk 1 above)
Comment (jlai-d2l) [reply tier1-icons.html]: No, but it happened as a result of `npm run build`, and so I thought it was normal?

File: images/tier3/notstarted-circle.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg width="30" height="30" xmlns="http://www.w3.org/2000/svg">
```
Comment (dlockhart) [inline images/tier3/notstarted-circle.svg]: Make sure you have a `viewBox="0 0 30 30"` on the `svg` element -- that's for IE.

File: tier3-icons.html (hunk 2 above)
Comment (jlai-d2l) [reply tier3-icons.html]: Happy to make whatever changes are necessary. But again, these were also part of the output of `npm run build`.

File: images/tier3/inprogress-circle.svg [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<svg width="30" height="30" xmlns="http://www.w3.org/2000/svg">
```
Comment (dlockhart) [inline images/tier3/inprogress-circle.svg]: I think an argument can be made that this icon should be mirrored in RTL mode by adding the `mirror-in-rtl="true"` attribute to the `svg` element.

From [Google's Material Design guidelines on RTL icons](http://google.github.io/material-design-icons/#icons-in-rtl):
> When text, layout, and iconography are mirrored to support right-to-left UIs, anything that relates to time should be depicted as moving from right to left

File: tier3-icons.html (hunk 2 above)
Comment (dbatiste) [reply tier3-icons.html]: Yeah ideally I think we don't want them.  I'm just not too sure why there are there now.

File: tier1-icons.html (hunk 1 above)
Comment (dlockhart) [reply tier1-icons.html]: I think it's OK... probably a result of the last person _not_ using the generated script?

File: tier3-icons.html (hunk 2 above)
Comment (dlockhart) [reply tier3-icons.html]: Didn't Justin's commit remove the line feed characters?

**Comment by jlai-d2l** [comment]
Re: icon names, we are also using the `check-circle` icon, and so we named the icons to be consistent with the existing naming structure. 

The icons are meant to represent the different states of progress within a course as a quick visual signal to managers about the progress of direct reports on their teams: Not started, In Progress, and Complete. 

I'm not good at naming things, if there's a better name for these, happy to make changes.

**Comment by dlockhart** [comment]
OK cool. Yeah `check-circle` is a little bit of an oddball because it can mean so many things that just leaving it descriptive of what it is kind of works.

How about: `course-progress-in-progress`, `course-progress-not-started` and `course-progress-complete`?

**Comment by jlai-d2l** [comment]
No. We use `check-circle` as the icon for "Course Progress Complete." Did you want to keep the icon naming for the other 2?

**Comment by dlockhart** [comment]
Oh interesting. Hmm, in that case I might almost suggest creating a copy of `check-circle` and calling that `course-progress-complete`. The reason being... we might change `check-circle` in the future (color, size, etc.) and not realize that it's tightly related to the other `course-progress-` icons.

File: tier3-icons.html (hunk 2 above)
Comment (dbatiste) [reply tier3-icons.html]: Hmm yeah.  Maybe if last person didn't use generated, then it explains it.  :)

**Pull request merged**